src/QuickMarcEditor/QuickMarcEditorRows/ContentField/ContentField.js

```diff
@@ -29,12 +38,19 @@ export const ContentField = ({
     }
   }, [ref, input.value]);
 
+  useEffect(() => {
+    if (ref.current && onProcessSubfieldRef) {
```

Comment:
Why is processing refs needed inside this component, instead of on `<Field>` component level?

Reply:
The `inpurtRef` prop in `TextArea` overrides the `inputRef` prop passed from the `<Field>` level.
```
<TextArea
  {...props} <----
  input={input}
  inputRef={ref} <----
  data-testid={id}
  onFocus={processSubfieldFocus}
/>
```

Reply:
It can also be implemented to avoid writing `ref` processing in the `ContentField`.

```
export const ContentField = ({
  input,
  id,
  inputRef,
  ...props
}) => {
  const ref = useRef();

  const getRef = (reference) => {
    if (reference) {
      ref.current = reference;
      inputRef(reference);
    }
  };

  const processSubfieldFocus = useCallback(({ target }) => {
    const end = target.value.length;

    target.setSelectionRange(end, end);
  }, []);

  useLayoutEffect(() => {
    if (ref.current) {
      ref.current.style.height = '0';

      const resizeStyles = getResizeStyles(ref.current);

      Object.keys(resizeStyles).forEach(property => {
        ref.current.style[property] = resizeStyles[property];
      });
    }
  }, [ref, input.value]);

  return (
    <TextArea
      {...props}
      input={input}
      inputRef={getRef}
      data-testid={id}
      onFocus={processSubfieldFocus}
    />
  );
};
```

Reply:
But why do we need to handle refs at all here? This will move focus to content field when adding a new field, which is against requirements in [UIQM-261](https://issues.folio.org/browse/UIQM-261)
In scope of this ticket we just need to make sure that when a content field is focused - cursor is at the end of the input, and `onFocus` should be enough for this

Reply:
yes, you are right, for some reason I misunderstood, I will correct it now

Reply:
Corrected